backend/lib/adapters/gateways/okta/okta-gateway.ts

```diff
@@ -22,6 +22,40 @@ type OktaUserInfo = {
   email_verified: boolean;
 };
 
+type OktaJwtParseError = {
+  name: string;
+  message: string;
+  userMessage: string;
+  jwtString: string;
+  parsedHeader: Record<string, string | number | object>;
+  parsedBody: Record<string, string | number | object>;
+  innerError: {
+    errno: number;
+    code: string;
+    syscall: string;
+  };
+};
+
+function isJwtParseError(maybe: unknown): maybe is OktaJwtParseError {
+  return !!maybe && typeof maybe === 'object' && 'name' in maybe && maybe.name === 'JwtParseError';
+}
+
+async function verifyAccessTokenWithRetry(
+  issuer: string,
+  token: string,
+  audience: string,
+  retry = true,
+): Promise<object> {
+  try {
+    return await verifyAccessToken(issuer, token, audience);
+  } catch (originalError) {
+    if (retry && isJwtParseError(originalError) && originalError.innerError.code === 'ECONNRESET') {
+      return await verifyAccessToken(issuer, token, audience);
+    }
+    throw originalError;
+  }
+}
```

Comment:
@btposey, I wonder if we should try a few times with an exponential backoff. Since we're not seeing this behavior in other environments, I'm not sure I'm confident with a single retry.